gremconnect/response_test.go

```diff
@@ -77,7 +77,6 @@ var (
 {
     "requestId": "d2476e5b-b2bc-6a70-2647-3991f68ab415",
     "status": {
-        "message": "Some error",
```

Comment:
Changes in this file were removed [here](https://gitlab.com/kaleidoscope.root/grammes-gremlin/-/commit/58aecaa0988a60f068717bef400c984533914001)

should be put back imo